health/micro-ui/web/micro-ui-internals/example/src/UICustomizations.js

```
@@ -795,54 +795,142 @@
+                ) {
+                  const triggeredFromMain = "OPEN_MICROPLANS";
+                  const response = await Digit.Hooks.microplanv1.useCompleteSetUpFlow({
+                    tenantId,
+                    microplanId,
+                    triggeredFrom: triggeredFromMain,
+                  });
+                  if (response && !response?.isError) {
+                    window.history.pushState(response?.state, "", response?.redirectTo);
+                    window.dispatchEvent(new PopStateEvent("popstate", { state: response?.state }));
+                  }
+                  if (response && response?.isError) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)